docs/protocol/core/v3.0.rst

```diff
@@ -808,7 +807,7 @@ compressed using gzip compression prior to storage::
         },
         "chunk_memory_layout": "C",
         "compressor": {
-            "codec": "http://purl.org/zarr/spec/codec/gzip",
+            "codec": "https://purl.org/zarr/spec/codec/gzip",
```

Comment:
Update to include version number in URI, as per codec spec draft?

Reply:
Sounds good, I'll include that in the gzip codec PR